src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -189,8 +189,17 @@ export class NetCoreDebugHelper implements DebugHelper {
         // Get Container Name if missing
         const containerName: string = debugConfiguration.containerName ?? await this.getContainerNameToAttach();
 
+        let debuggerPath: string = debugConfiguration.netCore?.debuggerPath;
+
         // If debugger path is not specified, install the debugger

```

Comment:
Nit: I'd modify this comment to say: ".... install the debugger if it doesn't exist in the container".